scripts/urban.js

```diff
@@ -6,16 +6,18 @@
 //
 
 module.exports = function (robot) {
-  robot.respond(/!?urban(.*)/i, function (res) {
+  robot.respond(/!?urban( .+)?/i, function (res) {
```

Comment:
I think this should this be `/!?urban (.+)/i`, removes the whitespace and makes input non-optional :)

Reply:
But then it wouldn't match and go through to the help text if someone uses it without args

Reply:
ah yep true :)

Reply:
Yeah, I spent a good hour learning regex to get it to do the thing properly. :) Figured we'd want it to respond to no input. If I have time/remember (and you guys approve) I'll go back over the other scripts and get them to do the same thing.